### ISSUE TITLE
Laravel 10

### ISSUE THREAD

**Issue by lindyhopchris** [laravel-json-api/laravel#224]
To upgrade to Laravel 10 (before this package is tagged for 10), use the following:

```bash
composer config minimum-stability dev 
composer config prefer-stable true
composer require "laravel-json-api/laravel:^3.0" --no-update
composer require "laravel-json-api/testing:^2.0" --dev --no-update
composer up "laravel-json-api/*" cloudcreativity/json-api-testing
```

**Comment by lindyhopchris** [comment]
These are the only breaking changes at the moment - though I'm not planning any more. It's just a major version bump for new version of Laravel plus dropping PHP 7.4 and 8.0.

### Changed

- Upgraded to Laravel 10 and set minimum PHP version to `8.1`.
- **BREAKING** If using the `laravel-json-api/cursor-pagination` package, you now need to passed the schema's `id` field
  to the paginator's `make()` method. I.e. use `CursorPagination::make($this->id())`

### Fixed

- **BREAKING** [#190](https://github.com/laravel-json-api/laravel/issues/190) The JSON:API media type now needs to be
  sent in the `Accept` header for a delete resource request. Previously there was no checking of the `Accept` media
  type, so anything could be sent. This is incorrect as the JSON:API specification shows the `Accept` header as
  `application/vnd.api+json` for [delete resource requests.](https://jsonapi.org/format/#crud-deleting)

**Comment by lindyhopchris** [comment]
Released as `3.0.0`. To upgrade:

```bash
composer require laravel-json-api/laravel --no-update
composer require laravel-json-api/testing --dev --no-update
composer up "laravel-json-api/*" cloudcreativity/json-api-testing
```
